app/src/main/java/org/mozilla/fenix/tabstray/NavigationInteractor.kt

```diff
@@ -193,4 +202,13 @@ class DefaultNavigationInteractor(
 
         // TODO show successful snackbar here (regardless of operation succes).
     }
+
+    override fun onSyncedTabClicked(tab: SyncTab) {
+        metrics.track(Event.SyncedTabOpened)
```

Comment:
There are three places that tracks `Event.SyncedTabOpened` now.  Is this expected?  Or should we remove the other two places that track this event?  The other two places are `SyncedTabsInteractor` and `SyncedTabsLayout`

Reply:
Ah, the references are in the old tabs tray code which will go away when we remove it in https://github.com/mozilla-mobile/fenix/issues/19112